autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -30,7 +32,9 @@ const NavBar: React.SFC = function App() {
             className={classes.menuButton}
             color="inherit"
             aria-label="menu"
-          />
+          >
+            <MenuIcon />
```

Comment:
You have to specify the icon that you want inside the button. I imported in the MenuIcon at the top of the file.

Reply:
Do we want this?

Reply:
Now that I think about it, I don't know if a navigation drawer is really necessary in this app. The only 2 routes are the landing page and the schedule page, so we might just delete the `<IconButton>` altogether. I'll let Gannon confirm which direction we want to go, though.

Reply:
Yeah honestly I don't think we'll need a navigation drawer, it should be that selecting a term brings you to the schedule view most likely.